digester_test.go

```diff
@@ -29,6 +29,9 @@ func (s *testSQLDigestSuite) TestNormalize(c *C) {
 		expect string
 	}{
 		{"SELECT 1", "select ?"},
+		{"select null", "select ?"},
+		{"select \\N", "select ?"},
+		{"SELECT `null`", "select null"},
```

Comment:
Should be ```select `null` ```.

Reply:
I will fix the test case once #1151 is merged.